src/utils/get-testcafe-version.ts

```diff
@@ -1,3 +1,19 @@
-export default function (): string {
-    return require('../../package.json').version;
+import logEntry from './log-entry';
+import debug from 'debug';
+
+const LOGGER = debug('testcafe:version');
+
```

Comment:
All debug loggers should be placed in the single place - https://github.com/DevExpress/testcafe/blob/master/src/utils/debug-loggers.ts.
Rewrite it as follows:
```js
const versionLogger = testcafeLogger.extend('version');
```